apps/form-management-app/src/app/components/styled-components.ts

```diff
@@ -0,0 +1,122 @@
+import styled from 'styled-components';
+
+export const BackButtonWrapper = styled.button`
```

Comment:
create a back button folder and add all files related to the button in one folder. and convert this to scss